backend/otel/otel.go

```diff
@@ -80,6 +81,64 @@ func projectToInt(projectID string) (int, error) {
 	return 0, e.New(fmt.Sprintf("invalid project id %s", projectID))
 }
 
+func getLogRow(ctx context.Context, ts time.Time, lvl, projectID, sessionID, traceID, spanID string, excMessage string, resourceAttributes, eventAttributes map[string]any) *clickhouse.LogRow {
```

Comment:
Would it be helpful to add more things to `NewLogRow`? This seems like a lot parameters that would be easy to mix the order.

Was thinking something like 

```go
clickhouse.NewLogRow(
  WithTimestamp(ts),
  WithProjectId(projectId),
  WithTraceId(traceId),
 ...
)
```

We can have it also handle errors too (for validating that the project id is valid).

Reply:
cleaned this up!